server/accounts.go

```diff
@@ -758,6 +764,13 @@ type remoteLatency struct {
 // sendLatencyResult will send a latency result and clear the si of the requestor(rc).
 func (a *Account) sendLatencyResult(si *serviceImport, sl *ServiceLatency) {
 	si.acc.mu.Lock()
+
+	sl.Type = ServiceLatencyType
+	a.srv.mu.Lock()
+	sl.ID = a.srv.nextEventID()
+	a.srv.mu.Unlock()
```

Comment:
@derekcollison so this is unfortunate, maybe add a nuid to account as well? Ugh.

Reply:
yea maybe..